pallets/settlement/src/lib.rs

```diff
@@ -1082,15 +1082,15 @@ impl<T: Config> Module<T> {
                     sender, receiver, ..
                 } => {
                     portfolios_pending_approval.insert(*sender);
-                    if T::Portfolio::skip_portfolio_affirmation(receiver, &ticker) {
+                    if T::Portfolio::skip_portfolio_affirmation(receiver, &asset_id) {
                         portfolios_pre_approved.insert(*receiver);
                     } else {
                         portfolios_pending_approval.insert(*receiver);
                     }
                 }
                 Leg::OffChain { .. } => continue,
             }
-            let asset_mediators = MandatoryMediators::<T>::get(ticker);
+            let asset_mediators = MandatoryMediators::<T>::get(asset_id);
```

Comment:
This is checked for off-chain legs too.  I don't think it makes sense.  Off-chain legs should only be fore assets that don't exist on-chain.